catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/content/data/impl/ContentItemImpl.java

```diff
@@ -141,11 +141,9 @@ public ContentItemImpl(
     }
     try {
       this.mimeType = new MimeType(this.mimeTypeRawData);
+      uri = new URI(CONTENT_SCHEME, this.id, this.qualifier);
     } catch (MimeTypeParseException e) {
       LOGGER.debug("Unable to create MimeType from raw data {}", mimeTypeRawData);
-    }
-    try {
-      uri = new URI(CONTENT_SCHEME, this.id, this.qualifier);
```

Comment:
Why was this logic changed? If we are unable to set `this.mimeType` do we still want to continue to set `this.uri`?